dask/dataframe/methods.py

```diff
@@ -51,6 +51,8 @@ def boundary_slice(df, start, stop, right_boundary=True, left_boundary=True,
                    kind='loc'):
     """Index slice start/stop. Can switch include/exclude boundaries.
 
+    Example
+    -------
```

Comment:
FYI I think that sphinx only renders certain titles "Examples" is rendered, but "Example" is not.  This doesn't matter in this case because this function probably won't be part of sphinx docs, but just wanted to share the general tip.